server/settings.py

```diff
@@ -69,7 +69,7 @@ def env(variable, fallback_value=None):
 
 INSTALLED_APPS = ["planning"]
 
-MODULES.append("planning")
+MODULES.extend(["planning", "planning.content_api"])
```

Comment:
I believe this is used by the Web API app, that is why you're getting `already registered error` in `server.planning.module.resources`

Reply:
This is where you'll need a separate config for the ContentAPI (this new config can be for tests only)